<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GameChoice2Item</class>
 <widget class="QWidget" name="GameChoice2Item">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>100</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="MyButtonGameChoice" name="pb_gameChoiceItem_2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>400</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="lb_fubiao_left">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="lb_fubiao_right">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>0</y>
     <width>100</width>
     <height>100</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MyButtonGameChoice</class>
   <extends>QPushButton</extends>
   <header>mybuttongamechoice.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
